### ISSUE TITLE
Bug in calib3D py_calibration tutorial

### ISSUE THREAD

**Issue by ngbusca** [opencv/opencv#22360]
The camera 3D calibration tutorial in py_calibration has a small bug: better chessboard corners are computed via `cornerSubPix` but the original corners are kept instead of the newly computed:

**Original code**:
``` python
    corners2 = cv.cornerSubPix(gray,corners, (11,11), (-1,-1), criteria)
    imgpoints.append(corners)
```

Should be:

``` python
    corners2 = cv.cornerSubPix(gray,corners, (11,11), (-1,-1), criteria)
    imgpoints.append(corners2) ### corrected line
```

**Comment by asmorkalov** [comment]
Thanks for the good catch! You are right.
HTML version of the tutorial: https://docs.opencv.org/4.x/dc/dbb/tutorial_py_calibration.html
Source file: py_calib3d/py_calibration/py_calibration.markdown
The issue should be fixed in 3.4 and then could be merged from 3.4 to 4.x

**Comment by Bhavil-13** [comment]
Hi, I'm new to open source, I wanted to contribute to OpenCV, is this issue resolved or can I do something about it? I have never done open source.
